### DISCUSSION TITLE
Sensory Connections between Sibernetic and c302?

### DISCUSSION THREAD

**Discussion by SSigurdson** [openworm/OpenWorm#331] [Q&A]
Hello!

I noticed that in the overarching project description/connections from https://docs.openworm.org/projects/ that sensory activation signals between Sibernetic and c302 had not yet been developed as of 2016. However, there has been development since then, so I was wondering if this has been developed yet in any way? I could not find it in my initial foray through the impressive array of repositories, but that could simply be due to their complexity so I wanted to confirm.

Thanks!

**Comment by slarson** [comment]
Check the discussion in this reference:

c302: a multiscale framework for modelling the nervous system of
Caenorhabditis elegans | Philosophical Transactions of the Royal Society B:
Biological Sciences
https://royalsocietypublishing.org/doi/10.1098/rstb.2017.0379#.Y7-ZeTTm5Gk.twitter

And the supplemental materials in this one:

https://royalsocietypublishing.org/doi/10.1098/rstb.2017.0376

Thanks for your question and interest!

**Reply by SSigurdson** [reply]
Thanks for the reply and pointing me in the direction of these useful resources! 

After having taken a look, it seems like the system is at a point where custom (user specified) input currents can be provided (either at the neuronal or muscular level), and proprioceptive feedback exists. However, sensory feedback from Sibernetic such as mechanical or thermal sensory signals have yet to be fed back into c302. Does that seem about right?
